recipes/vulkan-validationlayers/all/conanfile.py

```diff
@@ -57,18 +61,21 @@ def _get_compatible_spirv_tools_version(self):
         }.get(str(self.version), False)
 
     def requirements(self):
-        self.requires("spirv-tools/{}".format(self._get_compatible_spirv_tools_version), private=True)
+        #TODO set private=False, once the issue is resolved https://github.com/conan-io/conan/issues/9390
+        self.requires("spirv-tools/{}".format(self._get_compatible_spirv_tools_version), private=not self._is_cross_building())
         self.requires("vulkan-headers/{}".format(self.version))
         if tools.Version(self.version) >= "1.2.173":
-            self.requires("robin-hood-hashing/3.11.1")
+            self.requires("robin-hood-hashing/3.11.3")
         if self.options.get_safe("with_wsi_xcb") or self.options.get_safe("with_wsi_xlib"):
             self.requires("xorg/system")
         if self.options.get_safe("with_wsi_wayland"):
-            self.requires("wayland/1.18.0")
+            self.requires("wayland/1.19.0")
 
     def validate(self):
         if self.options["spirv-tools"].shared:
             raise ConanInvalidConfiguration("vulkan-validationlayers can't depend on shared spirv-tools")
+        if tools.is_apple_os(self.settings.os) and self.settings.arch in ["armv8", "armv8_32", "armv8.3"]:
+            raise ConanInvalidConfiguration("vulkan-validationlayers can't be build with this arch; open a PR to fix")
```

Comment:
What's wrong here? I can cross-build this recipe to armv8 if private=False

Reply:
Removed Exception